src/ape/_cli.py

```diff
@@ -28,17 +29,20 @@ def display_config(ctx, param, value):
 
     click.echo("# Current configuration")
 
+    access = import_module("ape.utils.basemodel").ManagerAccessMixin
```

Comment:
- [ ] lazy loading ManagerAccessMixin can be undone everywhere i think, it ends up getting loaded in anyway